fe1-web/features/social/screens/SocialSearch.tsx

```diff
@@ -3,13 +3,14 @@ import {
   FlatList, ListRenderItemInfo, StyleSheet, View, ViewStyle,
 } from 'react-native';
 import { useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
 
 import STRINGS from 'res/strings';
-import { makeCurrentLao, makeLastRollCallAttendeesList } from 'store';
+import { makeCurrentLao } from 'store';
 import { PublicKey } from 'model/objects';
 import TextBlock from 'components/TextBlock';
 import { gray } from 'styles/colors';
-import PropTypes from 'prop-types';
+import { makeRollCallAttendeesList } from 'features/events/reducer/EventsReducer';
```

Comment:
at some point, I assume this will shift to `features/rollcall` 😄

Reply:
Yes, that's what I think too.